services/search/utils.py

```diff
@@ -1,7 +1,38 @@
-import requests  # pylint: disable=import-error
+import logging
+from requests_futures.sessions import FuturesSession  # pylint: disable=import-error
 import constants
 
 
+class SynonymExtractor:
+    """
+    Wrapper class for requests_futures methods
+    """
+    def __init__(self, max_synonyms):
+        self.max_synonyms = max_synonyms
+        self.session = FuturesSession()
+
+    def create_conc_reqs(self, word_arr):
+        """
+        create an asynchronous request list using requests-futures
+        Attributes:
+            word_arr: A list of words which each is a different request
+            session: A requests-futures session
+        Return:
+            A list of requests for every word
+        """
+        reqs = []
+        for word in word_arr:
+            reqs.append(self.session.get(constants.SPANISH_API_URL+word))  # pylint: disable=no-member
```

Comment:
I'd recommend calling `req_to_json()` on the return of `self.session.get` before appending it to reqs. That way you can keep the function private to the class & avoid making the caller call `req_to_json` too.

Also, I think`reqs` here is really `responses`.